proto/ignition/msgs/joint_trajectory_point.proto

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+syntax = "proto3";
+package ignition.msgs;
+option java_package = "com.ignition.msgs";
+option java_outer_classname = "JointTrajectoryPointProtos";
+
+/// \ingroup ignition.msgs
+/// \interface JointTrajectoryPoint
+/// \brief Message for a single point in joint trajectory, used by joint_trajectory.proto
+
+import "ignition/msgs/duration.proto";
+
+message JointTrajectoryPoint
+{
+  repeated double positions     = 1;
```

Comment:
We're trying to document all the new messages we add, mind adding a `brief` for each field and for the message itself? The same wording as the ROS message would be fine.

Reply:
Added in https://github.com/ignitionrobotics/ign-msgs/pull/106/commits/3fda6a255bd15638126635b5252d08ad9ec5d3c3